assets/aml-benchmark/components/src/aml_benchmark/batch_inference_preparer/endpoint_data_preparer.py

```diff
@@ -17,11 +17,15 @@ class EndpointDataPreparer:
     PAYLOAD_HASH = "payload_id"
     PAYLOAD_GROUNDTRUTH = "label"
 
-    def __init__(self, model_type: str, batch_input_pattern: str, label_key: str = None):
+    def __init__(self, model_type: str,
+                 batch_input_pattern: str,
+                 label_key: str = None,
+                 additional_columns: str = None):
         """Init for endpoint data preparer."""
         self._model = OnlineEndpointModel(model_type=model_type, model=None, model_version=None)
         self._batch_input_pattern = batch_input_pattern
         self._label_key = label_key
+        self._additional_columns = additional_columns.split(",") if additional_columns else None
```

Comment:
Can we also trim spaces from ends after splitting to handle scenarios like "a, b, c , d" to be parsed as  ["a", "b" ,"c", "d"]? Also can we handle scenario of empty strings like "a,,b" which will become ["a", "", "b"] instead it should be ["a", "b"]?